src/main/java/io/meeds/qa/ui/pages/SpaceHomePage.java

```diff
@@ -1780,7 +1780,7 @@ private ElementFacade kudosButtonNumberToCommentActivityElement() {
   }
 
   private ElementFacade kudosButtonToCommentActivityElement() {
-    return findByXPathOrCSS("//div[contains(@class,'white border-radius')][1]//div[contains(@class,'v-list flex')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
+    return findByXPathOrCSS("//div[contains(@class,'white card-border-radius')][1]//div[contains(@class,'v-list flex')]//div[contains(@class,'d-inline-flex')][1]//div[@role='button']//span[@class='v-btn__content'][last()]");
```

Comment:
Let's attempt to fix this CSS selector and make it more generic and stable.

Reply:
done